app/src/test/java/com/rootstrap/android/SignInActivityViewModelTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.rootstrap.android
+
+import com.rootstrap.android.network.managers.session.SessionManager
+import com.rootstrap.android.network.managers.user.UserManager
+import com.rootstrap.android.network.models.User
+import com.rootstrap.android.network.models.UserSerializer
+import com.rootstrap.android.test.TestDispatcherProvider
+import com.rootstrap.android.test.UnitTestBase
+import com.rootstrap.android.ui.activity.main.SignInActivityViewModel
+import com.rootstrap.android.ui.activity.main.SignInState
+import com.rootstrap.android.util.NetworkState
+import com.rootstrap.android.util.extensions.ApiException
+import com.rootstrap.android.util.extensions.Data
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
```

Comment:
We must be aware of the changes for when it stops being experimental and update the version.